datahub-web-react/src/app/entityV2/shared/tabs/Incident/AcrylComponents/IncidentEditor.tsx

```diff
@@ -47,7 +52,7 @@ export const IncidentEditor = ({
     const [isLoadingAssigneeOrAssets, setIsLoadingAssigneeOrAssets] = useState(true);
 
     const [isRequiredFieldsFilled, setIsRequiredFieldsFilled] = useState<boolean>(
-        mode === IncidentAction.VIEW ? !isFormValid : false,
+        mode === IncidentAction.VIEW ? isFormValid : false,
```

Comment:
IncidentAction.VIEW doesn't this mean that the user is VIEWING and not editing the incident?

Reply:
My understanding is VIEW = UPDATE / EDIT
and ADD = CREATE

That's where the confusion came from. Consider renaming these enum symbols for clarity. Either way, I'll ship to unblock because this logic appears okay